python/src/aiconfig/editor/server/server_utils.py

```diff
@@ -75,6 +75,7 @@ def convert_to_mode(cls, value: Any) -> ServerMode:  # pylint: disable=no-self-a
 
 @dataclass
 class ServerState:
+    cli_config_path: str = "aiconfig_cli.config.yaml"
```

Comment:
I don't want to use `cli`, especially since we already have an `aiconfig_cli.py`, it's overloaded. Let's use a more clear name instead like `settings_path`